packages/nextcloud/README.md

```diff
@@ -28,6 +28,10 @@ final client = NextcloudClient(
 Not all endpoints work with just HTTP Basic auth, so it is advised to use app passwords obtained either directly in the Web UI by the user or using the [login flow](https://docs.nextcloud.com/server/latest/developer_manual/client_apis/LoginFlow/index.html#login-flow-v2).  
 Some endpoints do not need any authentication at all or provide extended information when the request is optionally authenticated.
 
+**Note**
+Flutter applications may require additional configuration to make HTTP requests.
+https://pub.dev/packages/http
```

Comment:
```suggestion
Flutter applications may require [additional configuration](https://docs.flutter.dev/data-and-backend/networking#platform-notes) to make HTTP requests.
``
Instead of linking to the http it should just link to the same page that is referenced there.

Reply:
Well, copy-pasting without thinking...